CLA.md

```diff
@@ -13,64 +13,53 @@ It is important to read and understand this legal agreement.
 
 ## How do I sign?
 
-If your work is done as an employee of your company, contact your company's legal department and ask to be put on the list of approved contributors for the CNCF CLA. Below, we have included steps for "Corporation signup" in case your company does not have a company agreement and would like to have one.
+After creating your first Pull Request the linux-foundation-easycla bot will respond with information regarding your CLA status along with a link to sign the CLA.
 
-#### 1. Log in to the Linux Foundation ID Portal with GitHub
+<img width="1065" alt="EasyCLA bot" src="https://user-images.githubusercontent.com/69111235/152226443-f6fe61ee-0e92-46c5-b6ea-c0deb718a585.png">
 
-Click one of:
-  * [Individual signup](https://identity.linuxfoundation.org/projects/cncf) to
-  sign up as an individual or as an employee of a signed organization.
-  * [Corporation signup](https://identity.linuxfoundation.org/node/285/organization-signup)
-  to sign as a corporation representative and manage signups from your organization.
+#### 1. Authorize EasyCLA to read some of your GitHub information
 
-Once you get to the sign in form, click "Log in with GitHub":
+<img width="554" alt="GitHub EasyCLA Authorization" src="https://user-images.githubusercontent.com/69111235/152228712-7d22f9d0-9f3c-4226-9ee0-bacba4b47725.png">
 
-![CNCFCLA1](http://i.imgur.com/tEk2x3j.png)
+Click on the "Please click here to be authorized" link to navigate to the GitHub Authorize Linux Foundation: EasyCLA page. Then click Authorize LF-Engineering to give the Linux Foundation read-only access to list the email addresses associated with your GitHub account.
 
-#### 2. Create Linux Foundation ID Portal account
+#### 2. Select from the two types of contributor
 
-After entering the various names, click "Submit":
+<img width="1407" alt="EasyCLA" src="https://user-images.githubusercontent.com/69111235/152224818-1246453a-b086-4a57-9d14-c10d62ad438f.png">
 
-If you are signing up as an employee, you must use your official
-person@organization.domain email address as your default GitHub
-address or configure it as an alias of your LF identify.
 
+After authorizing EasyCLA, you will be redirected to a page to identify which type of contributor you are. 
+Select the most appropriate option:
+  * Individual Contributor: To sign up as an individual or as an employee of a signed organization.
+  * Corporate Contributor: To sign as a corporation representative and manage signups from your organization.
```

Comment:
```suggestion
  * Individual Contributor: You are contributing as yourself, and not as part of another organization.
  * Corporate Contributor: You are contributing on behalf of your employer or other organization.
```

One last thing^^:;;
Missed it yesterday when reviewing.

Reply:
done :)